pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -1947,6 +1947,9 @@ type BackupSpec struct {
 
 	// PriorityClassName of Backup Job Pods
 	PriorityClassName string `json:"priorityClassName,omitempty"`
+
+	// ExponentialBackoffRetryPolicy the exponential backoff retry policy, currently only valid for snapshot backup
+	ExponentialBackoffRetryPolicy ExponentialBackoffRetryPolicy `json:"exponentialBackoffRetryPolicy,omitempty"`
```

Comment:
I suggest remove 'Exponential' from name, because we shouldn't expose our implementation. We may use other backoff policy later

Reply:
ok, i agree.